diff --git a/src/util/ua_util.c b/src/util/ua_util.c
index f5f496415..c76db4f2a 100644
--- a/src/util/ua_util.c
+++ b/src/util/ua_util.c
@@ -26,6 +26,10 @@
 #include "../../deps/parse_num.h"
 #include "../../deps/libc_time.h"
 
+#if defined(UA_ARCHITECTURE_WIN32)
+#include <windows.h>
+#endif
+
 const char * attributeIdNames[28] = {
     "Invalid", "NodeId", "NodeClass", "BrowseName", "DisplayName", "Description",
     "WriteMask", "UserWriteMask", "IsAbstract", "Symmetric", "InverseName",
